robottelo/constants.py

```diff
@@ -149,7 +149,7 @@
 VALID_GPG_KEY_BETA_FILE = "valid_gpg_key_beta.txt"
 
 RPM_TO_UPLOAD = "which-2.19-6.el6.x86_64.rpm"
-SRPM_TO_UPLOAD = "which-2.19-6.el6.x86_64.src.rpm"
+SRPM_TO_UPLOAD = "which-2.19-6.el6.src.rpm"
```

Comment:
This constant is also used by `tests/foreman/cli/test_repository.py::test_positive_upload_content_srpm`. Is this change safe for that test?

Reply:
Yeah it's safe as name was incorrect at all  :)